runtime/jvmti/jvmtiStackFrame.c

```diff
@@ -440,17 +440,43 @@ jvmtiPopFrame(jvmtiEnv* env,
 		ENSURE_CAPABILITY(env, can_pop_frame);
 
 		rc = getVMThread(
-				currentThread, thread, &targetThread, JVMTI_ERROR_OPAQUE_FRAME,
-				J9JVMTI_GETVMTHREAD_ERROR_ON_NULL_JTHREAD | J9JVMTI_GETVMTHREAD_ERROR_ON_DEAD_THREAD | J9JVMTI_GETVMTHREAD_ERROR_ON_VIRTUALTHREAD);
+				currentThread, thread, &targetThread, JVMTI_ERROR_NONE,
+				J9JVMTI_GETVMTHREAD_ERROR_ON_NULL_JTHREAD | J9JVMTI_GETVMTHREAD_ERROR_ON_DEAD_THREAD);
```

Comment:
Is it legal to continue if its a dead thread?

Reply:
No. So, `JVMTI_ERROR_THREAD_NOT_ALIVE` will be returned by `getVMThread` when the `J9JVMTI_GETVMTHREAD_ERROR_ON_DEAD_THREAD` flag is set.